server/src/main/java/org/elasticsearch/index/mapper/DocumentParser.java

```diff
@@ -584,7 +585,29 @@ private static void parseDynamicValue(final ParseContext context, ObjectMapper p
         parseObjectOrField(context, mapper);
     }
 
+    public static void addFulltextFields(ParseContext context, FieldMapper fieldMapper) throws IOException {
+        DocumentMapper docMapper = context.docMapper();
+        Set<String> ftIndices = docMapper.mapping().ftIndices();
+        if (ftIndices != null) {
```

Comment:
also here, see my previous comment. please do not call this `fulltext`, a normal column also can have a fulltext index but does not have any sources.

Reply:
I got rid of FT mentions in https://github.com/crate/crate/pull/14026/commits/5971882565b9ef6e47dec5770bf5228b13346a39